app/main/views/frameworks.py

```diff
@@ -1187,3 +1187,27 @@ def view_contract_variation(framework_slug, variation_slug):
         agreed_details=agreed_details,
         supplier_name=supplier_name,
     ), 400 if form_errors else 200
+
+
+@main.route('/frameworks/<framework_slug>/opportunities', methods=['GET'])
+@login_required
+def opportunities_dashboard(framework_slug):
+    try:
+        framework = data_api_client.get_framework(slug=framework_slug)['frameworks']
+        supplier_framework = data_api_client.get_supplier_framework_info(
+            supplier_id=current_user.supplier_id,
+            framework_slug=framework['slug']
+        )['frameworkInterest']
+    except APIError as e:
+        abort(e.status_code)
+    if not (framework['framework'] == 'digital-outcomes-and-specialists' and supplier_framework['onFramework']):
+        abort(404)
+    opportunities = data_api_client.find_brief_responses(
```

Comment:
This list of opportunities needs to be filtered by the specific framework.  As this is the following URLs show me the same list of things - both DOS and DOS2 opportunities:
```
http://localhost/suppliers/frameworks/digital-outcomes-and-specialists/opportunities
http://localhost/suppliers/frameworks/digital-outcomes-and-specialists-2/opportunities
```

(There should probably be a test for this - i.e. that checks only opportunities in the framework explicitly mentioned in the URL are shown.)